<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1126</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="sidebar_menu" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>801</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(1,1,1)</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Bus Managment</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>241</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="routes_button">
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{
	background-color:white;
	color:black;
}
QPushButton{
color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;
border-radius: 10px;
margin-left: 10px;
margin-right:10px;
}
</string>
        </property>
        <property name="text">
         <string>Routes</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/point-objects-32.png</normaloff>:/icons/point-objects-32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bus_button">
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{
	background-color:white;
	color:black;
}
QPushButton{
color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;
border-radius: 10px;
margin-left: 10px;
margin-right:10px;
}
</string>
        </property>
        <property name="text">
         <string>Buses</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/bus-32.png</normaloff>:/icons/bus-32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drivers_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{
	background-color:white;
	color:black;
}
QPushButton{
color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;
border-radius: 10px;
margin-left: 10px;
margin-right:10px;
}
</string>
        </property>
        <property name="text">
         <string>Drivers</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/user-32.png</normaloff>:/icons/user-32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>690</y>
       <width>241</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="settings_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{
	background-color:white;
	color:black;
}
QPushButton{
color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;
border-radius: 10px;
margin-left: 10px;
margin-right:10px;
}
</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/settings-4-32.png</normaloff>:/icons/settings-4-32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:checked{
	background-color:white;
	color:black;
}
QPushButton{
color: rgb(255, 255, 255);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border: none;
border-radius: 10px;
margin-left: 10px;
margin-right:10px;
}
</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/icons/door-8-24.png</normaloff>:/icons/door-8-24.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="main_screen" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>871</width>
      <height>731</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>871</width>
       <height>721</height>
      </rect>
     </property>
     <widget class="QWidget" name="buses_page">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>300</y>
         <width>121</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Buses Page</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="routes_page">
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>300</y>
         <width>131</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Routes Page</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="drivers_page">
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>300</y>
         <width>141</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Drivers Page</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="header" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>871</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>20</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:20 px;
border: 1px solid gray;
border-radius: 10px;</string>
     </property>
     <property name="placeholderText">
      <string>Search...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;MS Shell Dlg 2&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Hi, Admin!</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(153, 153, 153);
</string>
     </property>
     <property name="text">
      <string>Welcome to Bus Managment</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
